lib/mbta.js

```diff
@@ -1,9 +1,34 @@
 'use strict';
 
-// Code here.
+// Dictionary of line names with arrays representing each subway line.
+const mbtaSystem = {
+  'Red':
+    ["Alewife", "Davis", "Porter", "Harvard", "Central", "Kendall", "Park Street", "South Station"],
+  'Green':
+    ["Kenmore", "Hynes", "Copley", "Arlington", "Boylston", "Park Street", "Government Center"],
+  'Orange':
+    ["Forest Hills", "Back Bay", "Chinatown", "Downtown Crossing", "State", "Park Street", "Haymarket", "North Station"]
+}
 
```

Comment:
Nice. Love how you did this.  Line as the key and an array of stops as the value.  Good job.